packages/boot/test/bootstrapTests/vaults-integration.test.ts

```diff
@@ -1,86 +1,85 @@
-import { makeWalletFactoryDriver } from '../../tools/drivers.js';
+import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
+import { makeAgoricNamesRemotesFromFakeStorage } from '@agoric/vats/tools/board-utils.js';
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { Fail } from '@endo/errors';
 
 // presently all these tests use one collateral manager
 const collateralBrandKey = 'ATOM';
 
-const likePayouts = (collateral, minted) => ({
-  Collateral: {
-    value: BigInt(collateral * 1_000_000),
-  },
-  Minted: {
-    value: BigInt(minted * 1_000_000),
-  },
+const likePayouts = (collateral: number, minted: number) => ({
+  Collateral: { value: BigInt(collateral * 1_000_000) },
+  Minted: { value: BigInt(minted * 1_000_000) },
 });
 
-const makeDefaultTestContext = async t => {
+const makeDefaultTestContext = async () => {
   console.time('DefaultTestContext');
-  const swingsetTestKit = await makeSwingsetTestKit(t.log);
 
-  const { runUtils, storage } = swingsetTestKit;
+  const storage = makeFakeStorageKit('bootstrapTests');
+
+  const { handleBridgeSend } = makeMockBridgeKit({
+    ibcKit: { handleOutboundMessage: () => String(undefined) },
+    storageKit: storage,
+  });
+  const swingsetTestKit = await makeCosmicSwingsetTestKit({
+    configSpecifier: '@agoric/vm-config/decentral-itest-vaults-config.json',
+    handleBridgeSend,
+  });
+
+  const { EV, queueAndRun } = swingsetTestKit;
+
   console.timeLog('DefaultTestContext', 'swingsetTestKit');
-  const { EV } = runUtils;
 
   // Wait for ATOM to make it into agoricNames
   await EV.vat('bootstrap').consumeItem('vaultFactoryKit');
   console.timeLog('DefaultTestContext', 'vaultFactoryKit');
 
   // has to be late enough for agoricNames data to have been published
-  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(
-    swingsetTestKit.storage,
-  );
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
   agoricNamesRemotes.brand.ATOM || Fail`ATOM missing from agoricNames`;
   console.timeLog('DefaultTestContext', 'agoricNamesRemotes');
 
   const walletFactoryDriver = await makeWalletFactoryDriver(
-    runUtils,
+    { EV, queueAndRun },
     storage,
     agoricNamesRemotes,
   );
   console.timeLog('DefaultTestContext', 'walletFactoryDriver');
 
   console.timeEnd('DefaultTestContext');
 
-  return { ...swingsetTestKit, agoricNamesRemotes, walletFactoryDriver };
+  return {
+    ...swingsetTestKit,
+    agoricNamesRemotes,
+    storage,
+    walletFactoryDriver,
+  };
 };
 
 const test = anyTest as TestFn<
   Awaited<ReturnType<typeof makeDefaultTestContext>>
 >;
 
-test.before(async t => {
-  t.context = await makeDefaultTestContext(t);
-});
-test.after.always(t => {
-  return t.context.shutdown && t.context.shutdown();
-});
+test.before(async t => (t.context = await makeDefaultTestContext()));
+test.after.always(t => t.context.shutdown());
```

Comment:
Please revert these kinds of changes; using a block body rather than an expression communicates to the reader (and guarantees for the caller) that there is no relevant return value.
```suggestion
test.before(async t => {
  t.context = await makeDefaultTestContext(t);
});
test.after.always(t => {
  return t.context.shutdown && t.context.shutdown();
});
```
(e.g., `console.log([() => { "ignored"; }, () => (globalThis.foo = "returned")].map(fn => fn()))` logs `[undefined, "returned"]`)